case_study/memory_leaks/images_1/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Jun 23 08:50:38 CEST 2017
```

Comment:
are these gradle file changes expected?

Reply:
No, looks like it must have happened when I pushed my branch up for the first time. commit sha: 0fc4219890ac1730868a08db68ce6bce403161b9. I'll revert.